onnxruntime/python/tools/transformers/benchmark_gpt2.py

```diff
@@ -76,26 +76,73 @@ def onnxruntime_inference(ort_session, input_ids, past=None, total_runs=100):
 
     return ort_outputs, average_latency
 
+def onnxruntime_inference_with_binded_io(ort_session, input_ids, last_state, last_state_shape, past=None, present=None, present_shape=None, total_runs=100):
+    # Bind inputs and outputs to onnxruntime session        
+    io_binding = ort_session.io_binding()
+    # Bind inputs
+    io_binding.bind_input('input_ids', input_ids.device.type, 0, numpy.longlong, 
```

Comment:
It is better to use numpy.int64 to be consistent with onnx data type.

Reply:
For some reason numpy.int64 will result in "RuntimeError: No corresponding Numpy type for Tensor Type." error here. It seems like int64 type is matched to numpy.longlong in onnxruntime/python/onnxruntime_pybind_mlvalue.cc

Reply:
I see. Then let's keep longlong.